infra/feast-operator/api/v1alpha1/featurestore_types.go

```diff
@@ -118,8 +122,40 @@ type RegistryPersistence struct {
 }
 
 // RegistryFilePersistence configures the file-based persistence for the registry service
+// +kubebuilder:validation:XValidation:rule="(!has(self.pvc) && has(self.path)) ? self.path.startsWith('/') : true",message="Ephemeral stores must have absolute paths."
+// +kubebuilder:validation:XValidation:rule="(has(self.pvc) && has(self.path)) ? !self.path.startsWith('/') : true",message="PVC path must be a file name only, with no slashes."
 type RegistryFilePersistence struct {
-	Path string `json:"path,omitempty"`
+	Path      string     `json:"path,omitempty"`
+	PvcConfig *PvcConfig `json:"pvc,omitempty"`
+}
```

Comment:
`OnlineStoreFilePersistence` and `RegistryFilePersistence` are identical. Any reason we shouldn't make a single `FilePersistence` struct instead?

Reply:
Registry also supports the `s3_additional_kwargs` options (which will go under the `Registry.Persistence.File` section). Will we add it to a new PR dedicated to object store settings.

Reply:
ok, so this is in prep for future work... got it, thanks